### PR TITLE
Attempt to fix some issues identified in #2369

### PR DESCRIPTION
**Pull request by jtanx**
#2369

### CODE REVIEW

File: fontforge/svg.c [hunk 1]
```diff
@@ -755,7 +755,7 @@ static void svg_scdump(FILE *file, SplineChar *sc,int defwid, int encuni, int vs
 	c = LigCnt(sc->parent,best,univals,sizeof(univals)/sizeof(univals[0]));
 	fputs("unicode=\"",file);
 	for ( i=0; i<c; ++i )
-	    if ( univals[i]>='A' && univals[i]<'z' )
+	    if ( univals[i]>='A' && univals[i]<='z' )
```
Comment (frank-trampe) [inline fontforge/svg.c]: That's funny!

Was this silently and mysteriously breaking something for you?

File: fontforge/svg.c [hunk 2]
```diff
@@ -3356,7 +3358,7 @@ return( NULL );
 		xmlStrcmp(kids->name,(const xmlChar *) "missing-glyph")==0 )
 	    ++cnt;
     }
-    if ( sf->descent==0 ) {
+    if ( !has_font_face ) {
```
Comment (frank-trampe) [inline fontforge/svg.c]: That does seem much more reasonable.

**Comment by frank-trampe** [comment]
I like it.

File: fontforge/svg.c (hunk 1 above)
Comment (jtanx) [inline fontforge/svg.c]: Not particularly, but it did end up encoding strings like `snoo&#x7a;e` instead of `snooze`

**Pull request merged**